core/src/components/popover/popover.tsx

```diff
@@ -449,6 +449,9 @@ export class Popover implements ComponentInterface, PopoverInterface {
     const { el } = this;
 
     const { inline, delegate } = this.getDelegate(true);
+
+    this.ionMount.emit();
```

Comment:
It might be beneficial to add a comment to why it's important to have `ionMount` above `attachComponent` to prevent it being moved to a lower position.

Reply:
Good call! Added in https://github.com/ionic-team/ionic-framework/pull/28072/commits/c4beade7437f373d139fc02fca4953f4c4fc9394.